ADAL/src/ADALConstants.m

```diff
@@ -31,6 +31,7 @@
 NSString *const ADAL_BROKER_HASH_KEY                 = @"hash";
 NSString *const ADAL_BROKER_INTUNE_RESPONSE_KEY      = @"intune_mam_token";
 NSString *const ADAL_BROKER_INTUNE_HASH_KEY          = @"intune_mam_token_hash";
+NSString *const ADAL_BROKER_NONCE_KEY                = @"broker_nonce";
```

Comment:
nit: This now seems that it is being used at multiple places, may be we should move it to MSID constants.

Reply:
discussed offline, 4.0.* will have it in common core.